packages/devtools_app/test/navigation_test.dart

```diff
@@ -37,7 +35,7 @@ void main() {
     testWidgets(
         'Generates a route name with parameters with an empty route in the context',
         (WidgetTester tester) async {
-      String generatedRoute;
+      late String generatedRoute;
```

Comment:
why are some of these late and some are nullable? I would think they should all be late or all be nullable